qubesmanager/global_settings.py

```diff
@@ -245,7 +245,7 @@ def __init_updates__(self):
             self.updates_dom0_val =\
                 not os.path.isfile(self.dom0_updates_file_path)
 
-        self.updates_dom0.setChecked(self.updates_dom0_val)
+        self.updates_dom0.setChecked(bool(self.updates_dom0_val))
```

Comment:
This should be converted to `bool` earlier. Otherwise check in `__apply_updates__` will fail (bool != str).

Reply:
BTW you can add issue reference: QubesOS/qubes-issues#4988